Gruntfile.js

```diff
@@ -305,6 +305,26 @@ module.exports = function (grunt) {
         }())
       }
     },
+  
+    search: {
+      noAbsoluteUrlsInCss: {
+        files: {
+          src: ['assets/sass/**/*.scss']
+        },
+        options: {
+          searchString: /url\([\'\"]\//g,
+          failOnMatch: true,
+          logFile: SCSSLINT_OUT_FILE,
+          logFormat: 'xml',
+          onMatch: function(match) {
+            grunt.log.errorlns('URLs starting with \'/\' are not allowed in SCSS files. ');
+            grunt.log.errorlns('Found in file: ' + match.file + '. Line: ' + match.line);
```

Comment:
nit: I think it'd be nice to add a line of how to fix it, i.e. "Fix this by replacing with a relative link"